src/main/java/uk/gov/hmcts/reform/em/orchestrator/financialremedyservice/FinancialRemedyBundlePopulator.java

```diff
@@ -26,12 +26,12 @@ public JsonNode populateNewBundle(JsonNode caseData) {
         JsonNode scanned4BFormDocument = caseData.findValue("uploadScanned4BForm");
         JsonNode miniFormADocument = caseData.findValue("miniFormA");
 
-        JsonNode additionalDocuments = caseData.findValue("uploadAdditionalDocument");
-
         addDocumentToBundle(ccdBundleDTO, scanned4AFormDocument, "scanned4AFormDocument");
         addDocumentToBundle(ccdBundleDTO, scanned4BFormDocument, "scanned4BFormDocument");
         addDocumentToBundle(ccdBundleDTO, miniFormADocument, "miniFormADocument");
 
+        JsonNode additionalDocuments = caseData.findValue("uploadAdditionalDocument");
```

Comment:
This was a checkstyle thing about additionalDocuments, which should be initiated near its use